wagtail/contrib/styleguide/templates/wagtailstyleguide/base.html

```diff
@@ -256,12 +256,12 @@ <h3>Colour signifiers</h3>
 
             <h4>Positive</h4>
 
-            <a href="#" class="button yes">yes</a>
+            <a href="#" class="button button-small yes">yes</a>
             <a href="#" class="button button-small yes">yes</a>
 
             <h4>Negative</h4>
 
-            <a href="#" class="button no">No</a>
+            <a href="#" class="button button-small no">No</a>
```

Comment:
can I double check this is deliberate? previously we had the regular size and the small size. Or, the link (`<a>`) and button (`<button>`) versions. this and the above are essentially 2 duplicate lines

Reply:
Good point. But all other buttons above are the link and button variants of the same thing. This row was the outlier and caused confusion as I thought the styles were broken. 

I'll do a PR with a new row for the variants of non-small here for yes/no with Paarth though. 

Cc @PaarthAgarwal

Reply:
Can we also add type="button" in that PR?

Reply:
@zerolab  do you mind taking a look https://github.com/wagtail/wagtail/pull/9136
FYI @PaarthAgarwal  - let me know if that all makes sense, also includes the border-color change discussed tonight.

Reply:
left a couple of notes. 👏 for you both for staying on top of this.